package.json

```diff
@@ -170,7 +170,7 @@
         "ansible.executionEnvironment.image": {
           "scope": "resource",
           "type": "string",
-          "default": "quay.io/ansible/ansible-devtools-demo-ee:v0.1.0",
+          "default": "quay.io/ansible/creator-ee:latest",
```

Comment:
Not sure about the "latest".

Reply:
I do not want to have to update it in at least 3 repositories every time we release a new one. We will need to add a feature to display its versioned tag at runtime, but when it comes to defaults, using latest is the only flexible version we have. At least of a long time we will only support use of latest container. Once we release a new one, 

PS. I hope that latest meaning is about `latest` tag meaning translates to latest tagged version, not every random merge made to main branch.